site/content/src/main/exercises/stdlib/EmptyValues.scala

```diff
@@ -0,0 +1,147 @@
+package stdlib
+
+import org.scalatest._
+
+/** Empty Values
+  *
+  * Empty Values Description
+  */
+
+object EmptyValues extends FlatSpec with Matchers with exercise.Section {
+
+
+  /** emptyValuesEmptyValues
+    *
+    * ==null==
+    *
+    * Scala's `null` is the same as in Java. Any reference type can be `null`, like Strings, Objects, or your own classes. Also just like Java, value types like Ints can't be `null`. 
+    *
+    * ==Null==
+    *
+    * [Null](http://www.scala-lang.org/api/current/index.html#scala.Null) is a trait whose only instance is `null`. It is a subtype of all reference types, but not of value types. It purpose in existing is to make it so reference types can be assigned `null` and value types can't. 
+    *
+    * ==Nothing==
+    *
+    * [Nothing](http://www.scala-lang.org/api/current/index.html#scala.Nothing) is a trait that is guaranteed to have _zero_ instances. It is a subtype of all other types. It has two main reasons for existing: to provide a return type for methods that **never** return normally (i.e. a method that always throws an exception). The other reason is to provide a type for Nil (explained below). 
+    *
+    * ==Unit==
+    *
+    * [Unit](http://www.scala-lang.org/api/current/index.html#scala.Unit) in Scala is the equivalent of `void` in Java. It's used in a function's signature when that function doesn't return a value.
+    *
+    * ==Nil==
+    *
+    * [Nil](http://www.scala-lang.org/api/current/index.html#scala.collection.immutable.Nil$) is just an empty list, exactly like the result of `List()`. It is of type `List[Nothing]`. And since we know there are no instances of Nothing, we now have a list that is statically verifiable as empty. Nice to have. 
+    *
+    * An empty list can be represented by another nothing value: *Nil*
+    */
+  def emptyValuesEmptyValues(res0: String) {
+    assert(List() === res0)
+
```

Comment:
All the exercises have a couple of empty lines in the end, can we get rid of it?